js/helpers/replaceAll.js

```diff
@@ -1,15 +1,29 @@
 var escapeRegExp = require( "lodash/string/escapeRegExp" );
+var forEach = require( "lodash/collection/forEach" );
 
 /**
  * Replace placeholders in the passed string.
- * @param {string} string The string that needs placeholders to be replaced.
+ * @param {object} obj The object to be used for replacements.
  * @param {object} replacements The replacement values to be used.
  * @returns {string} The string with the replaced placeholders.
  */
-var replaceAll = function( string, replacements ) {
-	var replacementString = escapeRegExp( Object.keys( replacements ).join( "|" ) ).replace( "\\|", "|" );
+var replaceAll = function( obj, replacements ) {
```

Comment:
This file should not be used anymore. Remove references after https://github.com/Yoast/YoastSEO.js/pull/321 has been merged.